gui/default/index.html

```diff
@@ -682,6 +682,14 @@ <h4 class="panel-title">
                       <th><span class="far fa-fw fa-clock"></span>&nbsp;<span translate>Uptime</span></th>
                       <td class="text-right">{{system.uptime | duration:"m"}}</td>
                     </tr>
+                    <tr>
+                      <th><span class="fas fa-fw fa-qrcode"></span>&nbsp;<span translate>Identification</span></th>
+                      <td class="text-right">
+                        <span tooltip data-original-title="{{'Click to see full identification string and QR code.' | translate}}">
+                          <a href="" style="color:inherit" ng-click="showDeviceIdentification(thisDevice())">{{deviceShortID(deviceCfg.deviceID)}}</a>
```

Comment:
We use link styling for clickable items in other rows, so I'd do the same here. Currently I don't see any hint that there's an action when clicking the short device ID.

Reply:
Done.